dataland-framework-toolbox/scripts/convert-excel-to-csv.sh

```diff
@@ -19,7 +19,9 @@ set -euxo pipefail
 # false - Remove spaces --> No
 # -1 - Export all sheets
 
-libreoffice --headless \
```

Comment:
Is this to make it run on windows machines? What do you guys think, would it have merit to look into the possibility to detect the operating system and set the corresponding path here?

Reply:
ja könnten wir theoretisch machen wenn wir zeit haben

Reply:
Can be closed when a coment is added to the file, how to use it under linux. Generall it should be distributed for windows.

Reply:
Comment is there - closed